tracer/test/test-applications/security/Samples.Security.AspNetCore5/Controllers/IastController.cs

```diff
@@ -965,5 +965,12 @@ static string CopyStringAvoidTainting(string original)
         {
             return new string(original.AsEnumerable().ToArray());
         }
+        
+        [HttpGet("VerbTampering")]
+        [HttpPut("VerbTampering")]
+        public IActionResult VerbTampering()
```

Comment:
AFAICT, you've only tested this with MVC in ASP.NET Core 5+. I would strongly suggest you test it with .NET Core 2.1/3.0 (as they used a different routing model) as well as minimal APIs...